README.md

```diff
@@ -46,17 +46,17 @@ send it to a file (or stdout).
 GoFlow2 is a wrapper of all the functions and chains thems.
```

Comment:
```suggestion
GoFlow2 is a wrapper of all the functions and chains what is this?.
```

Reply:
Not sure what I meant when I wrote this :/